src/components/bookForm/BookForm.jsx

```diff
@@ -27,6 +30,13 @@ function BookForm({ createBook }) {
             value={title}
             className="book__form__input"
           />
+          <input
+            type="text"
+            placeholder="Author"
+            onChange={(e) => setAuthor(e.target.value)}
+            value={author}
+            className="book__form__input"
+          />
```

Comment:
Nice job so far👍🏼. The add and remove functionalities work properly, but there is just a slight issue😉. when you try to remove any of the books that are initialized, it removes all of them at once. kindly do well to fix that. you can take a look at the video below😉.

https://www.loom.com/share/b955298ff3e548ce836f9c5182228ef4?sid=2373343e-7af7-48f3-b101-900228a42115